src/Data/Cache.hs

```diff
@@ -190,23 +190,23 @@ lruCache c = Cache
           else do
             makeRoom
             lift $ writeCache c k v
```

Comment:
I don't understand why you removed the dollar from the first line but not the second. Aren't the functions easier to read and understand when they follow the same pattern?

Also, the `c` is again no longer aligned with anything.